yarn-project/circuits.js/package.json

```diff
@@ -22,7 +22,7 @@
     "clean": "rm -rf ./dest .tsbuildinfo",
     "formatting": "run -T prettier --check ./src && run -T eslint ./src",
     "formatting:fix": "run -T prettier -w ./src",
-    "remake-bindings": "DEBUG=wasm ts-node-esm src/cbind/circuits.in.ts && prettier -w src/cbind/circuits.gen.ts",
+    "remake-bindings": "DEBUG=wasm ts-node-esm src/cbind/circuits.in.ts && sed -i 's/CallStackItem/PublicCallStackItem/g' src/cbind/circuits.gen.ts && prettier -w src/cbind/circuits.gen.ts ",
```

Comment:
👀

Reply:
Fixing this one

Reply:
This isnt a major blocker btw @ludamad so no pressure!